src/modals/LogInModal.tsx

```diff
@@ -104,7 +104,7 @@ class LogInModal extends Component<ModalProps, ModalStates> {
                             </div>
                         </div>
                         <div className="buttons">
-                            {!config.networks.includes('Mainnet') ? 
+                            {!config.networks.includes('xxxx') ? 
```

Comment:
you should never change this file, please change again with Mainnet. Everytime you change LoginModal.tsx and config, remember to discard the changes to those files before pushing